Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM gliderlabs/alpine:3.4
+FROM golang:alpine AS build
+
+COPY . /go/src/github.com/Zipcar/lambda-resource
+
+RUN cd /go/src/github.com/Zipcar/lambda-resource && \
+    go build -o /go/src/github.com/Zipcar/lambda-resource/bin/lambda-resource-linux-amd64
+
+
+FROM alpine:edge AS resource
 
 RUN apk add --no-cache ca-certificates
 
-ADD bin/lambda-resource-linux-amd64 /opt/resource/out
+COPY --from=build /go/src/github.com/Zipcar/lambda-resource/bin/lambda-resource-linux-amd64 /opt/resource/check
+
+RUN cd /opt/resource && \
```

Comment:
you can combine this RUN with the previous RUN to collapse one of the layers - faster downloading in concourse, better caching in garden